### DISCUSSION TITLE
Weekly and Monthly views

### DISCUSSION THREAD

**Discussion by jedahan** [tenJirka/reGenda#6] [Ideas]
Would be nice to have a weekly view, monthly view, probably more landscape oriented.

**Comment by tenJirka** [comment]
Landscape orientation is a problem. As you can tell from the readme, reGenda is based on [simple](https://rmkit.dev/apps/sas) and as far as I know there is no support for orientation change now.

The weekly view should be possible, I'm not so sure about the monthly one, more precisely I'm worried about readability with so much information on the screen.
I'll look into it, but not anytime soon I'm afraid.
